udata/core/activity/api.py

```diff
@@ -61,6 +61,12 @@
     help="Filter activities for that particular organization",
     location="args",
 )
+activity_parser.add_argument(
+    "related_to",
+    type=str,
+    help="Filter activities for that particular reuse or dataset id",
```

Comment:
It can currently also be organization. But in any case, it is generic and could be applied to dataservices & co in the future.
```suggestion
    help="Filter activities for that particular object id (ex : reuse, dataset, etc.)",
```